.github/workflows/rust.yml

```diff
@@ -35,21 +35,26 @@ jobs:
           toolchain: "1.70.0"
           components: clippy, rustfmt
       - uses: actions/checkout@v3
+      - name: Set Environment
+        run: >
+          export RUSTFLAGS="-D warnings" ;
+          export APP_ENVIRONMENT="test"
       - name: Rust format
         run: >
           "$HOME/.cargo/bin/cargo" fmt --manifest-path rust_dev_preview/cross_service/Cargo.toml --all --check &&
           "$HOME/.cargo/bin/cargo" fmt --manifest-path rust_dev_preview/examples/Cargo.toml --all --check &&
+          "$HOME/.cargo/bin/cargo" fmt --manifest-path rust_dev_preview/lambda/Cargo.toml --all --check
           "$HOME/.cargo/bin/cargo" fmt --manifest-path rust_dev_preview/webassembly/Cargo.toml --all --check
       - name: Rust lint
         if: success() || failure()
         run: >
-          "$HOME/.cargo/bin/cargo" clippy --manifest-path rust_dev_preview/cross_service/Cargo.toml -- -D warnings &&
-          "$HOME/.cargo/bin/cargo" clippy --manifest-path rust_dev_preview/examples/Cargo.toml -- -D warnings &&
-          "$HOME/.cargo/bin/cargo" clippy --manifest-path rust_dev_preview/webassembly/Cargo.toml -- -D warnings
+          "$HOME/.cargo/bin/cargo" clippy --manifest-path rust_dev_preview/cross_service/Cargo.toml --all &&
+          "$HOME/.cargo/bin/cargo" clippy --manifest-path rust_dev_preview/examples/Cargo.toml --all &&
+          "$HOME/.cargo/bin/cargo" clippy --manifest-path rust_dev_preview/lambda/Cargo.toml --all &&
+          "$HOME/.cargo/bin/cargo" clippy --manifest-path rust_dev_preview/webassembly/Cargo.toml --all
       - name: Rust test
         run: >
           "$HOME/.cargo/bin/cargo" test --manifest-path rust_dev_preview/cross_service/Cargo.toml --all &&
           "$HOME/.cargo/bin/cargo" test --manifest-path rust_dev_preview/examples/Cargo.toml --all &&
+          "$HOME/.cargo/bin/cargo" test --manifest-path rust_dev_preview/lambda/Cargo.toml --all &&
           "$HOME/.cargo/bin/cargo" test --manifest-path rust_dev_preview/webassembly/Cargo.toml --all
-        env:
```

Comment:
Any reason you used export instead of env?

Reply:
I didn't want to duplicate the stanza